dolphinscheduler-standalone-server/src/main/resources/application.yaml

```diff
@@ -90,22 +115,60 @@ registry:
 
 security:
   authentication:
-    # Authentication types (supported types: PASSWORD,LDAP)
-    type: PASSWORD
+    # Authentication types (supported types: PASSWORD,LDAP,CASDOOR_SSO)
+    type: CASDOOR_SSO
```

Comment:
You can modify this in local tests, but have to revert to `PASSWORD` when the PR is finalized before merging

Reply:
Yes, I will modify it back. THX!
The current version is just a draft for @EricGao888 to review.